pkg/master/master.go

```diff
@@ -174,6 +174,7 @@ func resetMetrics(w http.ResponseWriter, req *http.Request) {
 
 func (m *Master) InstallAPIs(c *Config) {
 	apiGroupsInfo := []genericapiserver.APIGroupInfo{}
+	autoscalingGroupVersion := unversioned.GroupVersion{Group: "autoscaling", Version: "v1"}
```

Comment:
move this closer to where it's used, and I'd recommend repeating it inside each if block.